### PR TITLE
Use chat as guest

### PR DESCRIPTION
**Pull request by dnaranjo89**
Improve the non registered user interact with the chat

### CODE REVIEW

File: web/templates/draws/display_draw.html [hunk 1]
```diff
@@ -101,24 +101,19 @@
             var user_name = "{{ user.pk }}";
         {% else %}
             var user_name = getCookie("user_name");
```
Comment (Mariocj89) [inline web/templates/draws/display_draw.html]: is this cookie used anymore?

File: web/templates/draws/display_draw.html [hunk 2]
```diff
@@ -101,24 +101,19 @@
...
-            "authorized": {%if user.is_anonymous%} Boolean(user_name) {% else %} true {% endif %},
             "is_enabled": {{bom.enable_chat|yesno:"true,false"}},
             "url_send_message": "{% url 'chat_add_message' %}",
             "user_name": user_name,
             "draw_id": "{{ bom.pk }}",
             "msg_type_your_message": "{% trans 'Type your message here...' %}",
-            "msg_login_first": "{% trans 'Please, log in to be able to use the chat'%}",
+            "msg_login_first": "{% trans "To use the chat you should <a href='#'>login</a> or <a href='#'>register</a>" %}",
+            "msg_alias": "{% trans "If not, simply type an Alias that allows others to recognize you" %}",
+            "msg_error_alias": "{% trans "The alias mus have between 2 and 20 characters" %}",
```
Comment (Mariocj89) [inline web/templates/draws/display_draw.html]: what about translating the messages? -.-

File: web/templates/draws/display_draw.html (hunk 1 above)
Comment (dnaranjo89) [inline web/templates/draws/display_draw.html]: Yes. The mechanism for saving the alias is the same, only the modal has been removed.

**Comment by dnaranjo89** [comment]
I had some problems compiling the translation in windows... I can try tomorrow again but if not I will let you know.

**Pull request merged**